### PR TITLE
doc: snap-schedule documentation

### PR DESCRIPTION
**Pull request by jan--f**
Signed-off-by: Jan Fajerski <jfajerski@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by jan--f** [comment]
jenkins render docs

**Comment by batrick** [comment]
> jenkins render docs

FYI, this is no longer necessary. You can see the rendered docs by clicking the "Details" link for the `docs/readthedocs.org:ceph` jenkins check.

the build is currently failing too, please take a look @jan--f

File: doc/mgr/snap_schedule.rst [hunk 1]
```diff
@@ -0,0 +1,177 @@
+.. _mgr-snap_schedule:
```
Comment (batrick) [inline doc/mgr/snap_schedule.rst:1]: Let's put this in the `doc/cephfs` tree. Then link to itt from the TOC.

**Review by batrick — CHANGES_REQUESTED** [review]

**Comment by batrick** [comment]
@jan--f ping

File: doc/cephfs/snap-schedule.rst [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+arguments. Once a keyword argument was encountered, all following arguments are
+assumed to be keyword arguments too.
+
+Snapshot schedules are identified by path, their repeat interval and their start
+time. The
+repeat interval defines the time between two subsequent snapshots. It is
+specified by a number and a period multiplier, one of `h(our)`, `d(ay)` and
+`w(eek)`. E.g. a repeat interval of `12h` specifies one snapshot every 12
+hours.
+The start time is specified as a time string (see DOC BELOW LINK). By default
```
Comment (batrick) [inline doc/cephfs/snap-schedule.rst:34]: `DOC BELOW LINK` todo item?

**Review by batrick — CHANGES_REQUESTED** [review]
otherwise LGTM, thanks!

File: doc/cephfs/snap-schedule.rst (hunk 2 above)
Comment (jan--f) [reply doc/cephfs/snap-schedule.rst]: oh yes good catch, thx and fixed.

**Review by batrick — APPROVED** [review]

**Pull request merged**